src/lib/containers/IconSvg.tsx

```diff
@@ -443,24 +466,35 @@ function IconMapping(props: { icon: string; sx?: SxProps }) {
       return <AlternateEmail sx={sx} />
     case 'restore':
       return <RestoreFromTrashTwoTone sx={sx} />
+    case 'label':
+      return <LabelTwoTone sx={sx} />
+    case 'upload':
+      return <UploadTwoTone sx={sx} />
+    case 'flag':
+      return <FlagTwoTone sx={sx} />
     default:
       return <></>
   }
 }
 
 function IconSvg(props: IconSvgProps) {
-  const { icon, label = '', sx } = props
+  const { icon, label = '', sx, wrap = true } = props
+
+  const Wrapper = wrap ? 'span' : React.Fragment
+  const wrapperProps = wrap
+    ? {
+        'data-svg': icon,
+        className: 'styled-svg-wrapper',
+        id: `icon-${icon}`,
+        role: 'img',
+      }
+    : {}
 
   return (
     <Tooltip placement="top" title={label}>
-      <span
-        data-svg={icon}
-        className="styled-svg-wrapper"
-        id={`icon-${icon}`}
-        role={'img'}
-      >
+      <Wrapper {...wrapperProps}>
```

Comment:
The `span` wrapper was causing problems with the MUI IconButton component, so let's make it optional.